### ISSUE TITLE
Should drop downs read the label when opened?

### ISSUE THREAD

**Issue by joe-allen-89** [adaptlearning/adapt-contrib-matching#190]
### Subject of the issue/enhancement/features
When tabbing through a page you're jumped to the select button, on opening the drop down list the screen reader reads out the option but not the label/question itself. From a usability point of view I think we should announce the label when opening the drop down to avoid any confusion. 

Wondering if anybody has any reason not to do this/any reasons we've not done it before?

**Comment by oliverfoster** [comment]
Sounds good if you can make it happen. LabelledBy similar to mcq etc?

**Comment by joe-allen-89** [comment]
> Sounds good if you can make it happen. LabelledBy similar to mcq etc?

Exactly, add an ID to the label and use labelledby in the drop down.
